src/frontend/apps/standalone_site/src/features/Contents/components/ContentsRouter/ContentsRouter.tsx

```diff
@@ -1,19 +1,20 @@
 import { Route, Switch } from 'react-router-dom';
 
 import { Text404 } from 'components/Text';
-import { Contents } from 'features/Contents';
 
-import routes from '../../routes';
+import useContentRoutes from '../../hooks/useContentRoutes';
 import { useContentFeatures } from '../../store/contentsStore';
+import Contents from '../Contents/Contents';
 
 const ContentsRouter = () => {
   const { featureRouter } = useContentFeatures((state) => ({
     featureRouter: state.featureRouter,
   }));
+  const routes = useContentRoutes();
 
-  const videoPath = routes.CONTENTS.subRoutes.VIDEO.path;
-  const classroomPath = routes.CONTENTS.subRoutes.CLASSROOM.path;
-  const webinarPath = routes.CONTENTS.subRoutes.LIVE.path;
+  const videoPath = routes.CONTENTS.subRoutes?.VIDEO?.path;
```

Comment:
What happen if it doesn't exists ?

Reply:
[fixup! ♻️(website) add useRoutes hook](https://github.com/openfun/marsha/pull/2253/commits/fd8ba6b23e23adda5a588b110e7f5535383ca238)